src/HelloWorldPrinter.java

```diff
@@ -10,4 +10,8 @@ public void printHelloWorld() {
     System.out.println("Hello world!");
   }
 
+  public void printHelloMessage(String name) {
```

Comment:
What happens if `name` is `null`? Should this corner case be handled?

Reply:
In our code this should never happen, but I've added a check for this just in case.